products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,184 @@
+---
+id: cli
+title: CLI
+pagination_label: CLI
+sidebar_label: CLI
+sidebar_position: 1
+sidebar_class_name: cli
+keywords: ['cli']
+description: The SailPoint CLI is a terminal-based tool you can use to interact with your IDN tenant.
+slug: /tools/cli
+tags: ['CLI']
+---
+
+## Start using the CLI
+
+The SailPoint CLI is a terminal-based tool you can use to to interact with your IdentityNow (IDN) tenant. The CLI provides a text-based environment you can use to run operations known as "commands" to interact with your tenant however you want.
+
+You can use the CLI to perform many functions you would have otherwise used Postman or custom scripts to perform before. For example, you can use the CLI to call the SailPoint APIs to do whatever you want in your IDN tenant, and you can do all this directly on the command line, with minimal setup.
+
+Learn how to use the SailPoint command line interface (CLI) in this guide.
+
+![Sail](./assets/img/vhs/Sail.gif)
+
+## Contents
+
+- [Start using the CLI](#start-using-the-cli)
+- [Contents](#contents)
+- [Requirements](#requirements)
+- [Get the CLI](#get-the-cli)
+  - [Homebrew](#homebrew)
+- [Configuration](#configuration)
+  - [Assisted configuration](#assisted-configuration)
+  - [Manual configuration](#manual-configuration)
+    - [Environment variable configuration](#environment-variable-configuration)
+- [Usage](#usage)
+- [GitHub](#github)
+- [Contribution](#contribution)
+- [Questions](#questions)
+
+## Requirements
+
+- Golang version 1.18 or above. You can download it [here](https://go.dev/dl/). You can run `go version` to check your version.
+
+## Get the CLI
+
+To install the SailPoint CLI: use a package manager for the OS of your choice. Prebuilt binaries for OS X, Windows, and Linux are provided in [Releases](https://github.com/sailpoint-oss/sailpoint-cli/releases).
+
+### Homebrew
+
+MacOS users can use [Homebrew](https://brew.sh/) to install the CLI. Run these commands in your terminal:
+
+```sh
+brew tap sailpoint-oss/tap && brew install sailpoint-cli
+```
+
+Then make sure you can run the `sail` command.
+
+## Configuration
+
+To configure the CLI to connect and authenticate to your IDN tenant, you must do the following:
+
+- Find your tenant name. To learn how to find it, refer to [Getting Started](https://developer.sailpoint.com/idn/api/getting-started#find-your-tenant-name). The CLI will use this tenant name to connect to your IDN instance.
```

Comment:
Hey @james-haytko-sp,

I am Absolutely blown away by the quality of your documentation, I adjusted some small things that have changed this week but these docs are fantastic, you did a really amazing job.

It looks like a lot of the links we are using in this markdown have full links to different parts of developer.sailpoint.com, we need to adjust these to path only links, that way these pages with build and work correctly in both development and preproduction environments like localhost and  identitysoon

I will give an example here, but we will need to update the link format where ever we have a local link in the markdown

This one for example instead of 
```
https://developer.sailpoint.com/idn/api/getting-started#find-your-tenant-name
```

it would be 
```
/idn/api/getting-started#find-your-tenant-name
```

Reply:
Thanks Luke I appreciate it. That's right I'll fix those links now.